tests/Files/AReallyBadFile.php

```diff
@@ -2,9 +2,9 @@
 namespace Instapro;
 class AReallyBadFile {
 protected string $string='';
-public function __construct(Foo $a, Bar $bar){}
+public function __construct(Foo $a, \Baz\Bar $bar){}
 public function getMethod():string{
-return $this->string;   
+return $this->string;
```

Comment:
Can you add back the whitespace at the end of this please? I think this was intentional